Packages/com.unity.inputsystem/InputSystem/Devices/Remote/RemoteInputPlayerConnection.cs

```diff
@@ -31,7 +31,7 @@ internal class RemoteInputPlayerConnection :
     {
         public static readonly Guid kNewDeviceMsg = new Guid("fcd9651ded40425995dfa6aeb78f1f1c");
         public static readonly Guid kNewLayoutMsg = new Guid("fccfec2b7369466d88502a9dd38505f4");
-        public static readonly Guid kNewEventsMsg = new Guid("34d9b47f923142ff847c0d1f8b0554d9");
+        public static readonly Guid kNewEventsMsg = new Guid("53546641df1347bc8aa315278a603586");
```

Comment:
Unrelated fix but so minor I didn't bother routing it in separately. This one was a copypaste mistake where I ended up inadvertently reusing a GUID from the docs. Which then someone discovered because another Unity system had done the same :D